widgets/Html/index.jsx

```diff
@@ -44,13 +54,26 @@ class Html extends Component {
     handleYouTube(this.htmlContainer);
```

Comment:
I didn't think about it yesterday, but this won't work any more when html widget changes.

I tried to fix it by adding a lifecycle hook to additionally execute `handleYouTube` on component did update.

```
componentDidUpdate() {
    handleYouTube(this.htmlContainer);
  }
```